app/controllers/db/items_controller.rb

```diff
@@ -0,0 +1,60 @@
+class Db::ItemsController < Db::ApplicationController
+  permits :name, :url, :tombo_image, :main
+
+  before_action :authenticate_user!
+  before_action :load_work, only: [:index, :new, :create, :edit, :update, :destroy]
+  before_action :load_item, only: [:edit, :update, :destroy]
+
+  def index
+    @items = @work.items
+  end
+
+  def new
+    @item = @work.items.new
+    authorize @item, :new?
+  end
+
+  def create(item)
+    @item = @work.items.new(item)
+    authorize @item, :create?
+
+    if @item.save_and_create_db_activity(current_user, "items.create")
+      redirect_to db_work_items_path(@work), notice: "作品画像を登録しました"
+    else
+      render :new
+    end
+  end
+
+  def edit
+    authorize @item, :edit?
+  end
+
+  def update(item)
+    authorize @item, :update?
+
+    @item.attributes = item
+    if @item.save_and_create_db_activity(current_user, "items.update")
+      redirect_to db_work_items_path(@work), notice: "作品画像を更新しました"
+    else
+      render :edit
+    end
+  end
+
+  def destroy(id)
```

Comment:
Unused method argument - `id`. If it's necessary, use `_` or `_id` as an argument name to indicate that it won't be used. You can also write as `destroy(*)` if you want the method to accept any arguments but don't care about them.